packages/create-discord-bot/template/Deno/src/events/index.ts

```diff
@@ -11,7 +11,7 @@ export type Event<T extends keyof ClientEvents = keyof ClientEvents> = {
 	 *
 	 * @param parameters - The parameters of the event
 	 */
-	execute(...parameters: ClientEvents[T]): Promise<void> | void;
+	execute(this: void, ...parameters: ClientEvents[T]): Promise<void> | void;
```

Comment:
Same question though, an event‘s execute could also have some reason to access `this.name` or once.